Dockerfile

```diff
@@ -7,25 +9,33 @@ ARG BASE_IMAGE=quay.io/centos/centos:stream9
 FROM $BASE_IMAGE AS ironic-builder
 
 ARG IPXE_COMMIT_HASH=119c415ee47aaef2717104fea493377aa9a65874
+ARG MAKEFLAGS="-j100"
```

Comment:
A 100 jobs, I don't think thats reasonable default.

Reply:
no need to set MAKEFLAGS at all